src/MiniExcel/OpenXml/ExcelOpenXmlSheetReader.cs

```diff
@@ -417,7 +439,19 @@ private static void SetCellsValueAndHeaders(object cellValue, bool useHeaderRow,
             }
         }
 
-        private void SetSharedStrings()
+        private static XmlReaderSettings GetXmlReaderSettings(bool async)
+        {
+            return new XmlReaderSettings
+            {
+                IgnoreComments = true,
+                IgnoreWhitespace = true,
+                XmlResolver = null,
+                Async = async,
+            };
+        }
+
+        [Zomp.SyncMethodGenerator.CreateSyncVersion]
+        private async Task SetSharedStringsAsync(CancellationToken ct = default)

```

Comment:
`SetSharedStrings` is only being called from constructor. It is not ideal if constructor calls I/O functions.

Suggest marking the constructor with [Obsolete] and introducing:

```cs
static ExcelOpenXmlSheetReader Create(Stream stream, IConfiguration configuration, bool isUpdateMode = true);
static Task<ExcelOpenXmlSheetReader> CreateAsync(Stream stream, IConfiguration configuration, bool isUpdateMode = true, CancellationToken ct = default);

```